changelog.d/20250519_100900_30907815+rjmello_deprecate_legacy_containers_sc_32334.rst

```diff
@@ -0,0 +1,16 @@
+Deprecated
+^^^^^^^^^^
+
+- Deprecated the following legacy container features:
+
+  - :attr:`Executor.container_id <globus_compute_sdk.Executor.container_id>` attribute
+  - :class:`ContainerSpec <globus_compute_sdk.sdk.container_spec.ContainerSpec>` class
+  - ``container_uuid`` argument to :meth:`Client.register_function <globus_compute_sdk.Client.register_function>` method
+  - :meth:`Client.get_container <globus_compute_sdk.Client.get_container>` method
+  - :meth:`Client.register_container <globus_compute_sdk.Client.register_container>` method
+  - :meth:`Client.build_container <globus_compute_sdk.Client.build_container>` method
+  - :meth:`Client.get_container_build_status <globus_compute_sdk.Client.get_container_build_status>` method
+
+  Container functionality has moved to the endpoint configuration. For more information, see our
+  `documentation <https://globus-compute.readthedocs.io/en/latest/endpoints/endpoints.html#containerized-environments>`_
+  on containerized environments.
```

Comment:
`s/see our/see the/`.  Or perhaps:
```reStructuredText
See the Compute endpoint `containerized environment documentation`_.

.. _containerized environment documentation: https://gl...
```
(The suggestion is the text _that_ is linked, not the inline vs reference.)